### PR TITLE
update listings, updated aliases, fixed spellings and added websites

### PR DESCRIPTION
**Pull request by roxcoldiron**
Updated the listings
Updated aliases to combine more of the listings, also fixed some spellings

### CODE REVIEW

File: site/data/data.json [hunk 1]
```diff
@@ -11645,17 +11967,28 @@
         "cashapp": "",
         "last4": "1894"
       },
+      {
+        "name": "Molly Stark",
```
Comment (vsalvino) [inline site/data/data.json]: Duplicate

File: site/data/data.json [hunk 2]
```diff
@@ -4238,6 +4374,20 @@
       "The Cleveland Bagel Co."
     ]
   },
+  {
+    "name": "Cleveland Bagel Company",
```
Comment (vsalvino) [inline site/data/data.json]: Should be alias for cleveland bagel above.

File: site/data/data.json [hunk 3]
```diff
@@ -19130,6 +19658,34 @@
...
+        "name": "Brittany Vail",
+        "paypal": "brittanyvail24",
+        "venmo": "BritVail24",
+        "cashapp": "",
+        "last4": "6202"
+      }
+    ]
+  },
+  {
+    "name": "Wagon Wheel Pub",
```
Comment (vsalvino) [inline site/data/data.json]: I'm guessing these could be aliases as well

File: site/data/data.json (hunk 1 above)
Comment (roxcoldiron) [reply site/data/data.json]: fixing

File: site/data/data.json (hunk 2 above)
Comment (roxcoldiron) [reply site/data/data.json]: adding

**Pull request merged**